data/hai/unfettered 0 prologue.txt

```diff
@@ -515,3 +517,1013 @@ mission "Unfettered Jump Drive Trading"
+			`	"We have friends outside of our space, civilians and others. You see, we would do anything to help those who are True, and a few Hai want to join us. I can only hope you are willing to help us reunite with our brothers and sisters from Hai space," he explains, apparently a bit desperate, considering he knows very little, if anything at all, about you.`
+			choice
+				`	"Of course. What do you need me to do?"`
+					goto action
+				`	"What do you mean, brothers and sisters from Hai space? Are you not different?"`
+					goto brethren
+				`	"Why do you need me to do it?"`
+			label whyme
+			`	"Well, normally they would be able to pay to come here, but the price now is quite high; too high, due to an increase in the amount of Hai wanting to come to our space. Even in normal times, the Fettered Hai make it difficult for us to go anywhere; as you have probably noticed, they have a fleet on the border between us. And not that many Hai captains are willing to come here. When they do, some charge large fees because of the obvious risks involved - for instance, getting killed in the crossfire. Others are more generous, of course, but it is still not enough.`
+			`	"Yet you somehow made your way here, probably also under fire from our own people, but, if you choose to assist us, I'll ensure you won't have any issues getting out of here. So, are you willing to help us?"`
+			choice
+				`	"Of course. What do you need me to do?"`
+					goto action
+				`	"This seems like a bad idea. Ask someone else."`
+					decline
+			label brethren
+			`	"Then you have heard about the stories of the Drak modifying our people..." he sighs and deflates a bit. "I do not believe in such things. What makes a Hai - or any other species - 'true' is in their heart. It will take them wherever they want to go." He pauses, thinking, and when he speaks again it is with deep conviction. "Even if we quarrel with the Elders, it does not mean we hate other Hai, nor do they hate us! We do it only because we must, in order to keep living our way. We can still communicate with our people, and eventually they will all be freed from the Elders, who cannot help but think in a box."`
+			choice
+				`	"Will this get me into any trouble with the Hai?"`
+				`	"What limits?"`
+					goto limits
+				`	"Why do you need me to do it anyway?"`
+					goto whyme
+			`	"Quite the contrary! They may not believe in our way of doing things, but they would never dare to restrict the movement of their own people. It would be against all they stand for. Even if we behave differently, they know we are still Hai and that we would never hurt Hai civilians." He seems confident when telling you this.`
```

Comment:
```suggestion
			`	"Quite the contrary! They may not believe in our way of doing things, but they would never dare to restrict the movement of their own people. It would be against all they stand for. Even if we behave differently, they know we are still Hai and that we do all we can to avoid hurting Hai civilians," He says with confidence.`
```